resources/views/maps/markers/edit.blade.php

```diff
@@ -60,6 +61,19 @@
                                 </a>
                             </li>
                         </ul>
+                    </div>  
+                    </div>  
+
+                    <div class="pull-left">
+                        <div class="btn-group">
+                                <a role="button" tabindex="-1" class="btn btn-dynamic-delete btn-danger" data-toggle="popover"
```

Comment:
There seems to be a tab indent too many here :)

Reply:
Changes implemented